website/events/api/v2/serializers/event.py

```diff
@@ -120,3 +120,37 @@ def _url(self, instance: Event):
 
     def _maps_url(self, instance):
         return create_google_maps_url(instance.map_location, zoom=13, size="450x250")
+
+
+class EventListSerializer(EventSerializer):
```

Comment:
This should probably be handled in the parent with `many=True`

Reply:
Other places in the code do this too.